core/trino-main/src/main/java/io/trino/execution/buffer/PartitionedOutputBuffer.java

```diff
@@ -242,6 +243,7 @@ public void setNoMorePages()
     @Override
     public void destroy()
     {
+        memoryManager.summarizeBufferUtilization();
```

Comment:
actually, if you call `void recordOutputBufferUtilization()` (additionally) as part of `memoryManager.getUtilizationHistogram()` then you won't need this extra call here.

Reply:
ok